### PR TITLE
Restructure outputs page according to feedback

### PR DESCRIPTION
**Pull request by paulapreuss**

### CODE REVIEW

**Comment by paulapreuss** [comment]
This PR is a lot of small changes but they mostly concert formatting so I'm hoping the chances of it to break something are not that big. It would be nice to test them out with the simulation but if that's not possible because of simulation server errors it will hopefully still be okay.

**Comment by paulapreuss** [comment]
I worked on [6641045](https://github.com/rl-institut/cp_nigeria_app/pull/105/commits/6641045146ae67b0693a7933ba219a8d2572aeda) to create the output page tables with ajax calls instead, for now it is only the case for the summary table but on Monday I will expand it to all other tables and also create the parameter file to input the help texts.

File: app/cp_nigeria/views.py [hunk 1]
```diff
@@ -1199,43 +1198,43 @@ def cpn_outputs(request, proj_id, step_id=STEP_MAPPING["outputs"], complex=False
...
-    }
-
-    help_texts = {
-        "Total investment costs": help_icon("All upfront investment costs"),
-        "Total equity": help_icon("Total equity help text"),
-        "Total grant": help_icon("Total grant help text"),
-        "Initial loan amount": help_icon("Initial loan needed to cover the initial investment costs"),
-        "Replacement loan amount": help_icon("Amount needed to replace the system components"),
-    }
+    # financial_kpis = ft.financial_kpis
```
Comment (Bachibouzouk) [inline app/cp_nigeria/views.py]: it is better to remove code which will not be used, we have version control to reverse the change if needbe

**Review by Bachibouzouk — APPROVED** [review]
Thanks for the work @paulapreuss it looks nice!

**Pull request merged**